22:46:09,853 WARN  eu.stratosphere.nephele.configuration.GlobalConfiguration     - Cannot load configuration: unknown element plugins
22:46:09,890 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Reading location of job manager from configuration
22:46:09,900 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Determined address of job manager to be cloud-11.dima.tu-berlin.de/130.149.21.15:6002
22:46:09,908 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Announcing connection information cloud-20 to job manager
22:46:09,973 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server listener on 6122: starting
22:46:09,973 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server Responder: starting
22:46:09,974 INFO  eu.stratosphere.nephele.ipc.Server                            - IPC Server handler 0 on 6122: starting
22:46:11,092 INFO  eu.stratosphere.nephele.taskmanager.bufferprovider.GlobalBufferPool  - Initialized global buffer pool with 32768 buffers with a size 32768 bytes each
22:46:11,112 INFO  eu.stratosphere.nephele.taskmanager.bytebuffered.ByteBufferedChannelManager  - Initialized byte buffered channel manager with sender-side spilling disabled
22:46:11,122 INFO  eu.stratosphere.nephele.instance.HardwareDescriptionFactory   - Found Tenured Gen pool (max: 24051843072, used: 0)
22:46:11,122 INFO  eu.stratosphere.nephele.taskmanager.TaskManager               - Initializing memory manager with 22528 megabytes of memory
22:46:46,720 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Build Triads (17/25)
22:46:46,723 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Build Triads (17/25)
22:46:50,384 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (135/200)
22:46:50,384 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (135/200)
22:46:50,386 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (135/200)
22:46:50,387 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (135/200)
22:46:50,968 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (131/200)
22:46:50,968 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (131/200)
22:46:50,970 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (131/200)
22:46:50,971 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (131/200)
22:46:51,052 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (129/200)
22:46:51,052 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (129/200)
22:46:51,053 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (129/200)
22:46:51,055 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (129/200)
22:46:51,081 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (132/200)
22:46:51,081 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (132/200)
22:46:51,083 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (132/200)
22:46:51,084 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (132/200)
22:46:51,112 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (130/200)
22:46:51,112 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (130/200)
22:46:51,361 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (130/200)
22:46:51,362 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (130/200)
22:46:51,769 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (136/200)
22:46:51,769 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (136/200)
22:46:51,771 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (136/200)
22:46:51,772 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (136/200)
22:46:51,807 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (134/200)
22:46:51,807 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (134/200)
22:46:51,808 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (134/200)
22:46:51,809 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (134/200)
22:46:52,207 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Close Triads (133/200)
22:46:52,208 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from STARTING to RUNNING for task Triangles (133/200)
22:46:52,210 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Start PACT code.: Close Triads (133/200)
22:46:52,212 INFO  eu.stratosphere.pact.runtime.task.DataSinkTask                - Start PACT code: Triangles (133/200)
22:47:19,800 INFO  eu.stratosphere.pact.runtime.task.RegularPactTask             - Finished PACT code.: Build Triads (17/25)
22:47:19,800 INFO  eu.stratosphere.nephele.execution.ExecutionStateTransition    - TM: ExecutionState set from RUNNING to FINISHING for task Build Triads (17/25)
22:49:24,875 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-8722645902418225291_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:48393 remote=/130.149.21.24:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:24,876 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8722645902418225291_1722 bad datanode[0] 130.149.21.24:45020
22:49:24,877 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8722645902418225291_1722 in pipeline 130.149.21.24:45020, 130.149.21.3:45020, 130.149.21.17:45020: bad datanode 130.149.21.24:45020
22:49:27,448 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-8121216302005057703_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:48391 remote=/130.149.21.24:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:27,449 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8121216302005057703_1722 bad datanode[0] 130.149.21.24:45020
22:49:27,450 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8121216302005057703_1722 in pipeline 130.149.21.24:45020, 130.149.21.16:45020, 130.149.21.18:45020: bad datanode 130.149.21.24:45020
22:49:29,060 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_6274425079901908121_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:48381 remote=/130.149.21.24:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:29,061 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_6274425079901908121_1722 bad datanode[0] 130.149.21.24:45020
22:49:29,061 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_6274425079901908121_1722 in pipeline 130.149.21.24:45020, 130.149.21.64:45020, 130.149.21.65:45020: bad datanode 130.149.21.24:45020
22:49:41,023 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_3912198760179161840_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:48378 remote=/130.149.21.24:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:41,024 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_3912198760179161840_1722 bad datanode[0] 130.149.21.24:45020
22:49:41,024 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_3912198760179161840_1722 in pipeline 130.149.21.24:45020, 130.149.21.33:45020, 130.149.21.17:45020: bad datanode 130.149.21.24:45020
22:49:45,091 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_4914272615213689605_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:48373 remote=/130.149.21.24:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:45,092 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_4914272615213689605_1722 bad datanode[0] 130.149.21.24:45020
22:49:45,093 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_4914272615213689605_1722 in pipeline 130.149.21.24:45020, 130.149.21.18:45020, 130.149.21.66:45020: bad datanode 130.149.21.24:45020
22:49:51,703 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_1477637186220144557_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:48388 remote=/130.149.21.24:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:49:51,704 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1477637186220144557_1722 bad datanode[0] 130.149.21.24:45020
22:49:51,705 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1477637186220144557_1722 in pipeline 130.149.21.24:45020, 130.149.21.21:45020, 130.149.21.18:45020: bad datanode 130.149.21.24:45020
22:50:09,448 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_5914858446695895701_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:48385 remote=/130.149.21.24:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:09,449 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5914858446695895701_1722 bad datanode[0] 130.149.21.24:45020
22:50:09,449 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5914858446695895701_1722 in pipeline 130.149.21.24:45020, 130.149.21.2:45020, 130.149.21.28:45020: bad datanode 130.149.21.24:45020
22:50:11,722 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-3388553264717152866_1722java.net.SocketTimeoutException: 69000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:48375 remote=/130.149.21.24:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:11,723 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-3388553264717152866_1722 bad datanode[0] 130.149.21.24:45020
22:50:11,723 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-3388553264717152866_1722 in pipeline 130.149.21.24:45020, 130.149.21.63:45020, 130.149.21.2:45020: bad datanode 130.149.21.24:45020
22:50:31,007 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-8722645902418225291_1739java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:52586 remote=/130.149.21.3:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:31,008 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8722645902418225291_1739 bad datanode[0] 130.149.21.3:45020
22:50:31,009 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8722645902418225291_1739 in pipeline 130.149.21.3:45020, 130.149.21.17:45020: bad datanode 130.149.21.3:45020
22:50:33,550 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-8121216302005057703_1751java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:48357 remote=/130.149.21.16:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:33,551 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8121216302005057703_1751 bad datanode[0] 130.149.21.16:45020
22:50:33,551 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-8121216302005057703_1751 in pipeline 130.149.21.16:45020, 130.149.21.18:45020: bad datanode 130.149.21.16:45020
22:50:35,163 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_6274425079901908121_1760java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:53149 remote=/130.149.21.64:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:35,165 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_6274425079901908121_1760 bad datanode[0] 130.149.21.64:45020
22:50:35,165 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_6274425079901908121_1760 in pipeline 130.149.21.64:45020, 130.149.21.65:45020: bad datanode 130.149.21.64:45020
22:50:47,121 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_3912198760179161840_1789java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:38013 remote=/130.149.21.33:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:47,123 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_3912198760179161840_1789 bad datanode[0] 130.149.21.33:45020
22:50:47,123 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_3912198760179161840_1789 in pipeline 130.149.21.33:45020, 130.149.21.17:45020: bad datanode 130.149.21.33:45020
22:50:51,200 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_4914272615213689605_1795java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:58231 remote=/130.149.21.18:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:51,202 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_4914272615213689605_1795 bad datanode[0] 130.149.21.18:45020
22:50:51,202 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_4914272615213689605_1795 in pipeline 130.149.21.18:45020, 130.149.21.66:45020: bad datanode 130.149.21.18:45020
22:50:57,837 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_1477637186220144557_1807java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:52988 remote=/130.149.21.21:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:50:57,838 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1477637186220144557_1807 bad datanode[0] 130.149.21.21:45020
22:50:57,839 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_1477637186220144557_1807 in pipeline 130.149.21.21:45020, 130.149.21.18:45020: bad datanode 130.149.21.21:45020
22:51:15,543 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_5914858446695895701_1848java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:47405 remote=/130.149.21.2:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:15,545 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5914858446695895701_1848 bad datanode[0] 130.149.21.2:45020
22:51:15,545 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_5914858446695895701_1848 in pipeline 130.149.21.2:45020, 130.149.21.28:45020: bad datanode 130.149.21.2:45020
22:51:17,823 WARN  org.apache.hadoop.hdfs.DFSClient                              - DFSOutputStream ResponseProcessor exception  for block blk_-3388553264717152866_1863java.net.SocketTimeoutException: 66000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/130.149.21.24:39456 remote=/130.149.21.63:45020]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:155)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:128)
	at java.io.DataInputStream.readFully(DataInputStream.java:195)
	at java.io.DataInputStream.readLong(DataInputStream.java:416)
	at org.apache.hadoop.hdfs.protocol.DataTransferProtocol$PipelineAck.readFields(DataTransferProtocol.java:124)
	at org.apache.hadoop.hdfs.DFSClient$DFSOutputStream$ResponseProcessor.run(DFSClient.java:2582)

22:51:17,824 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-3388553264717152866_1863 bad datanode[0] 130.149.21.63:45020
22:51:17,824 WARN  org.apache.hadoop.hdfs.DFSClient                              - Error Recovery for block blk_-3388553264717152866_1863 in pipeline 130.149.21.63:45020, 130.149.21.2:45020: bad datanode 130.149.21.63:45020
